services/common/src/constants/environment.ts

```diff
@@ -110,6 +111,7 @@ export function setupEnvironment(
   ENVIRONMENT.environment = environment || "development";
   ENVIRONMENT.flagsmithKey = flagsmithKey;
   ENVIRONMENT.flagsmithUrl = flagsmithUrl;
+  ENVIRONMENT.flagsmithLocal = flagsmithLocal;
```

Comment:
aaaand when it was missing it was coming in as "<FLAGSMITH_ENABLE_LOCAL_EVALUATION>". Needed to add it to this function and then pass it to it from ms/core